### PR TITLE
Adds occupation area column in match view for Mapa do Acolhimento

### PR DESCRIPTION
**Pull request by vivianedias**

### CODE REVIEW

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **lpirola**
You can assign the PR to them by writing `/assign @lpirola` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fbonde).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/nossas/bonde/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bnd-operator** [comment]
@vivianedias: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr | 476eb97a0e82ef76960edaaf7a0c77f49bf1b693 | [link](http://dashboard-jx.nossas.tech/namespaces/jx/pipelineruns/nossas-bonde-pr-1486-pr-84vfc) | `/test pr`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by lpirola — APPROVED** [review]

**Comment by bnd-operator** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[lpirola](https://github.com/nossas/bonde/pull/1486#pullrequestreview-716052915 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=nossas%2Fbonde).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/nossas/bonde/blob/main/OWNERS)~~ [lpirola]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
